### PR TITLE
feat: Extended action button

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "uuidjs": "4.2.1",
     "webrtc-adapter": "6.4.8",
     "wire-web-config-default-prod": "https://github.com/wireapp/wire-web-config-wire#v0.10.2-0",
-    "wire-web-config-default-staging": "https://github.com/wireapp/wire-web-config-default#v0.10.1-staging.0"
+    "wire-web-config-default-staging": "https://github.com/wireapp/wire-web-config-default#v0.10.1"
```
Comment (atomrc) [inline package.json]: This should not be here

File: src/page/template/panel/conversation-details.htm [hunk 2]
```diff
@@ -6,208 +6,174 @@
...
+            <div class="conversation-details__read-receipts" data-uie-name="label-1to1-read-receipts">
+              <div class="panel__info-text panel__info-text--head panel__info-text--margin-bottom" data-bind="text: hasReceiptsEnabled() ? t('conversationDetails1to1ReceiptsHeadEnabled') : t('conversationDetails1to1ReceiptsHeadDisabled')"></div>
+              <div class="panel__info-text panel__info-text--margin-bottom" data-bind="text: t('conversationDetails1to1ReceiptsFirst')"></div>
+              <div class="panel__info-text panel__info-text--margin" data-bind="text: t('conversationDetails1to1ReceiptsSecond')"></div>
+            </div>
+          <!-- /ko -->
+        </div>
+      <!-- /ko -->
+    </div>
+    <div class="panel__content" style="flex-shrink: 0">
```
Comment (atomrc) [inline src/page/template/panel/conversation-details.htm:175]: @AndyLnd maybe moving the `style` to a proper css declaration would be a good idea

File: src/page/template/panel/message-details.htm [hunk 3]
```diff
@@ -12,49 +12,45 @@
...
+          data-uie-name="message-details-read-tab"
           ></div>
+        <div
+          class="panel__tab"
+          data-bind="css: {'panel__tab--active': !isReceiptsOpen()}, click: clickOnLikes, text: likesTitle()"
+          data-uie-name="message-details-like-tab"
+        ></div>
+      </div>
+    <!-- /ko -->
+    <div class="panel__content" data-bind="fadingscrollbar" style="flex-grow: 1">
```
Comment (atomrc) [inline src/page/template/panel/message-details.htm:29]: Maybe move this one to a css declaration also.

File: src/page/template/panel/guest-options.htm [hunk 4]
```diff
@@ -5,42 +5,40 @@
       <div class="panel__header__title" data-bind="text: t('guestOptionsTitle')"></div>
       <close-icon class="right-panel-close icon-button" data-bind="clickOrDrag: onClose" data-uie-name="do-close"></close-icon>
     </div>
-    <div class="panel__content">
-      <div class="panel__content__scroll" data-bind="antiscroll: shouldUpdateScrollbar">
```
Comment (atomrc) [inline src/page/template/panel/guest-options.htm:9]: @AndyLnd could you check if this `shouldUpdateScrollbar` is still used and remove it from the view model if it is not.

File: src/page/template/panel/participant-devices.htm [hunk 5]
```diff
@@ -19,77 +19,75 @@
...
-                    data-bind="click: clickToResetSession, style: {display: isResettingSession() ? 'none' : 'initial'}, text: t('participantDevicesDetailResetSession')"
-                    data-uie-name="do-reset-session"></span>
-            </div>
+        <div class="participant-devices__verify">
+          <div class="slider" data-uie-name="do-toggle-verified">
+            <input class="slider-input" type="checkbox" name="toggle" id="toggle" data-bind="checked: selectedClient().meta.isVerified">
+            <label class="button-label" for="toggle" data-bind="click: clickToToggleDeviceVerification, text: t('participantDevicesDetailVerify')"></label>
+          </div>
+          <div class="participant-devices__actions">
+            <svg class="svg-theme spin" data-bind="style: {display : isResettingSession() ? 'initial' : 'none'}" width="16" height="16"
```
Comment (Yserz) [inline src/page/template/panel/participant-devices.htm:68]: Let's make an icon component out of it

**Pull request merged**